ReleaseTooling/Template/README.md

```diff
@@ -54,21 +54,30 @@ To integrate a Firebase SDK with your app:
 
    c. Double-click the setting, click the '+' button, and add `-ObjC`
 
-10. Drag the `Firebase.h` header in this directory into your project. This will
+10. Add the `-lc++` flag to **Other Linker Settings**:
```

Comment:
I'm wavering about making this conditional or not. I think this can be ok since we always need to link in C++ anyway for Analytics.

Reply:
Regarding Analytics, there is something odd going on since:

> Crashlytics's module.modulemap includes link "c++" and so does Analytics's. But, weirdly, this error does not surface when just depending on Analytics.

Reply:
Does Analytics actually have .mm files like Crashlytics does?

Reply:
I wonder if this is a side effect of Apple "fixing" https://github.com/firebase/firebase-ios-sdk/issues/11818

Reply:
> Does Analytics actually have .mm files like Crashlytics does?

Doesn't look like it.

Reply:
> I wonder if this is a side effect of Apple "fixing" #11818

+1

--- 

Ah, just learned something else:

Without `-lc++`, build will succeed if you import Analytics or Crashlytics in a source file within your app.

Reply:
I think it's reasonable to merge this fix since it looks like Crashlytics can work without needing to be imported into a source file.

Reply:
SGTM, we may be able to remove when we switch to dynamic libraries.